socs/agents/vantagepro2/drivers.py

```diff
@@ -58,6 +59,19 @@ def F_to_C(temp):
     return (temp - 32) * (5 / 9)
 
 
+def wind_chill(temp, wind):
+    """Function to calculate wind chill temperature. Only valid if temp < 50F.
+    Taken from https://www.calculator.net/wind-chill-calculator.html
+    If temp > 50F, need to use heat index instead...
```

Comment:
If this formula is not valid above 50F can you add a check for the input temperature and just return the input temp?

Just playing around with the function as is it looks like it might report strange numbers otherwise:
```
>>> # expect -5F windchill
>>> wind_chill(5, 5)
-4.6271537104386
>>>
>>> # what about at higher temps?
>>> wind_chill(105, 5)
112.82872791458931
```